Suko/Cells/SUKMovieToAnimeCollectionViewCell.h

```diff
@@ -0,0 +1,19 @@
+//
+//  SUKMovieToAnimeCollectionViewCell.h
+//  Suko
+//
+//  Created by Alice Zhang on 7/22/22.
+//
+
+#import <UIKit/UIKit.h>
+#import "SUKMovie.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface SUKMovieToAnimeCollectionViewCell : UICollectionViewCell
+@property (strong, nonatomic) SUKMovie *movie;
+@property (weak, nonatomic) IBOutlet UILabel *movieTitleLabel;
```

Comment:
You usually don't want to expose views internal to a class to other files. It's usually best practice to have a `configureWithModel` method that will take in a data model and internally configure the view given the model. You can also have a configure method with individual parameters but having a data model to structure them is cleaner.